microGallery_Android/core-ui/build.gradle.kts

```diff
@@ -14,13 +14,14 @@ android {
 
 dependencies {
     implementation(platform(libs.androidxComposeBom))
-
-
     implementation(libs.androidxComposeFoundation)
     implementation(libs.androidxComposeMaterial3)
     implementation(libs.androidxComposeUi)
     implementation("androidx.compose.ui:ui-text-google-fonts:1.8.2")
-
+    implementation("androidx.compose.material3:material3:1.4.0-alpha15")
```

Comment:
dependencies should be added in the [version catalog](https://developer.android.com/build/migrate-to-catalogs). See `libs.versions.toml` file